corehq/warehouse/loaders/__init__.py

```diff
@@ -0,0 +1,57 @@
+from corehq.warehouse.loaders.app_status import (
+    ApplicationStatusFactLoader,
+    AppStatusFormStagingLoader,
+    AppStatusSynclogStagingLoader,
+)
+from corehq.warehouse.loaders.application import (
+    ApplicationDimLoader,
+    ApplicationStagingLoader,
+)
+from corehq.warehouse.loaders.domain import (
+    DomainDimLoader,
+    DomainStagingLoader,
+)
+from corehq.warehouse.loaders.domain_membership import (
+    DomainMembershipDimLoader,
+)
+from corehq.warehouse.loaders.form import FormFactLoader, FormStagingLoader
+from corehq.warehouse.loaders.group import GroupDimLoader, GroupStagingLoader
+from corehq.warehouse.loaders.location import (
+    LocationDimLoader,
+    LocationStagingLoader,
+)
+from corehq.warehouse.loaders.synclog import (
+    SyncLogFactLoader,
+    SyncLogStagingLoader,
+)
+from corehq.warehouse.loaders.user import UserDimLoader, UserStagingLoader
+from corehq.warehouse.loaders.user_group import UserGroupDimLoader
+from corehq.warehouse.loaders.user_location import UserLocationDimLoader
+
+
+def get_loader_by_slug(slug):
+    loaders = [
+        DomainStagingLoader,
+        DomainDimLoader,
+        UserStagingLoader,
+        UserDimLoader,
+        GroupStagingLoader,
+        GroupDimLoader,
+        LocationStagingLoader,
+        LocationDimLoader,
+        ApplicationStagingLoader,
+        ApplicationDimLoader,
+        UserGroupDimLoader,
+        UserLocationDimLoader,
+        DomainMembershipDimLoader,
+
```

Comment:
nit: might be worth adding a comment here instead of just a blank space to give a title for the category of loaders being grouped together. Helpful to understand the reason behind the grouping, because it is not obvious here for someone who does not know about the architecture around warehouse well.